includes/admin/upgrades/upgrade-functions.php

```diff
@@ -337,7 +337,6 @@ function give_show_upgrade_notices( $give_updates ) {
 			'callback' => 'give_v20_rename_donor_tables_callback',
 			'depend'   => [
 				'v20_move_metadata_into_new_table',
-				'v20_logs_upgrades',
```

Comment:
Just making sure I think through this. If we're removing the dependency, is it still necessary to mark it as complete in a migration?

Reply:
@JasonTheAdams is required. We store new migration registration data in the options table (legacy logic for background update) and then use that data to process migration, so removing `v20_logs_upgrades ` migration in code will fix the issue for plugin users who will upgrade from Give<=2.0 in the future but for the existing user, we have to mark `v20_logs_upgrades` migration complete in the database to resume migration process.